search/ranker.cpp

```diff
@@ -123,48 +116,44 @@ ftypes::Type GetLocalityIndex(feature::TypesHolder const & types)
   }
 }
 
-/// Makes continuous range for tokens and prefix.
-template <class TIter, class TValue>
-class CombinedIter
+// TODO: Format street and house number according to local country's rules.
+string FormatStreetAndHouse(ReverseGeocoder::Address const & addr)
 {
-  TIter m_cur;
-  TIter m_end;
-  TValue const * m_val;
-
-public:
-  CombinedIter(TIter cur, TIter end, TValue const * val) : m_cur(cur), m_end(end), m_val(val) {}
+  ASSERT_GREATER_OR_EQUAL(addr.GetDistance(), 0, ());
```

Comment:
We don't use distance in the function, why do we need to ASSERT something about it here?

Reply:
I have no idea.
Doesn't seem to be a victim of numerous refactorings, either: https://github.com/mpimenov/omim/commit/b87699ccc8b9ee23690b1ded15a96378497fe7e8

Reply:
removed